src/Map/index.js

```diff
@@ -184,14 +254,19 @@ export default class Map extends Component {
     }
 
     this.setState({
-      addresses: geocodedCoordinates,
-      isLoaded: true,
-      isLoading: false,
+      dataAddresses: geocodedCoordinates,
+      isDataAddressesLoaded: true,
+      isDataAddressesLoading: false,
     })
   }
 
-  getFilteredAddresses = () => {
```

Comment:
There's a comment at line 249 that refers to this `getFilteredAddresses` function, so if we're renaming this function we should probably update that comment.